service/src/main/java/com/epam/esm/service/dto/UserDto.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.esm.service.dto;
+
+import org.springframework.hateoas.RepresentationModel;
+
+import java.math.BigDecimal;
+import java.util.HashSet;
+import java.util.Set;
+
+public class UserDto extends RepresentationModel<UserDto> {
+  private Long id;
+  private String login;
+  private BigDecimal account;
```

Comment:
account?

Reply:
I got it. I changed the name of this field to balance. (fixed)